homeassistant/components/airgradient/quality_scale.yaml

```diff
@@ -0,0 +1,2 @@
+select:
```

Comment:
What does `select` or `include` mean in this context?
If they are a list of requirements, I prefer `implemented` (or `satisfied`)

Should we also have `exempted` (or `skip`...) to differentiate from `implemented`?

For complex requirements, it may also be useful to have some kind of `pending` (`in progress`...) if an item is partially implemented

Reply:
I think it makes sense to list the ones that we have implemented and only mention the ones we can't implement because of reasons.

And looking at how the rules are organized, I don't think there will be a need for a pending tag. You either implemented it or you did not. If only some entities have an unique_id, you did not satisfy the rule.

I do like the satisfy word, but I feel like if you are exempt from a rule, you also satisfied that rule in a way.